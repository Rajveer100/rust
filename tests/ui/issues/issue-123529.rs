trait Fun {
    pub fn test() {}
    //~^ ERROR visibility qualifiers are not permitted here
    //~| NOTE trait items always share the visibility of their trait
    //~| HELP remove the qualifier
}

fn main() {}
