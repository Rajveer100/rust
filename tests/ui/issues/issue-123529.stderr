error[E0449]: visibility qualifiers are not permitted here
  --> $DIR/issue-123529.rs:2:5
   |
LL |     pub fn test() {}
   |     ^^^
   |
   = help: remove the qualifier
   = note: trait items always share the visibility of their trait

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0449`.
